.github/ISSUE_TEMPLATE/ticket_template1.md

```diff
@@ -0,0 +1,27 @@
+🚨Please review the [guidelines for feature ticket](../HOWTOWRITEATICKETS.md) to this repository if unsure about what to write..
+
++## Problem / Motivation
+Describe your issue here. You can you the user story format.
+- As a ... Who?
+- I would like ... what?
+- So that ... why?
+
+
++## Proposed solution and tasks
+We could build this.... and it will solve that...
+
+List of tasks for this ticket to be complete:
+- [ ] A descriptive of a task
+- [ ] A description
+
++## Clarification needed
+These questions or assumptions need to be answered for a developer to complete this ticket. Delete this section if everything is clear.
+- [ ] A descriptive of a task
+- [ ] A description
+
++## Technical notes
+- We could use this technology 
+- More notes or suggestions
+
++## User interface / Wireframes
+Include wireframes if relevant.
```

Comment:
not just wireframes but also screenshots or renderings of anything that resembles the end goal?